src/pages/Lend/LoanChannelCategoryPage.vue

```diff
@@ -100,7 +100,7 @@ export default {
 				{
 					vmid: 'og:title',
 					property: 'og:title',
-					content: title ?? this.meta.title
+					content: title ?? `${this.meta.title} | Invest & Support`
```

Comment:
" | Invest & Support" should be added to `this.meta.title`, not just the og:title and twitter:title.